SpaceXAPI.Interfaces/Services/SpaceXAPIService.cs

```diff
@@ -14,17 +10,15 @@ namespace SpaceXAPI.Core.Services
     public class SpaceXAPIService : ISpaceXAPIService
     {
         private readonly HttpClient _client;
-        private readonly SpaceXAPIData clientData;
 
-        public SpaceXAPIService(IOptions<SpaceXAPIData> options)
+        public SpaceXAPIService(HttpClient client)
         {
-            clientData = options.Value;
-            _client = new HttpClient();
+            _client = client;
         }
 
         public async Task<ICollection<Launchpad>> GetLaunchpads()
         {
-            var response = await _client.GetAsync(new Uri(clientData.URL));
+            var response = await _client.GetAsync("/v2/launchpads");
```

Comment:
Note: I would move this relative path to the appsettings file, but for this exercise, I didn't have time to refactor the configuration object to account for it, so I went with hardcoding for now.